### PR TITLE
COG-1220 Disable hyperlinks for Phone Number 

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**
#### Which User story does this relate to (link)?
-  https://osi-cwds.atlassian.net/browse/COG-1220

#### Brief feature description
- Disables hyperlinks for the phone number in add user/ user detail page.

### CODE REVIEW

File: app/javascript/views/userDetail/UserDetail.test.js [hunk 1]
```diff
@@ -87,6 +87,12 @@ describe('UserDetail', () => {
     })
   })
 
+  describe('meta tag to disable hyperlinks for the phone number', () => {
+    it('verifies the meta tag exists', () => {
+      expect(wrapper.find('meta').length).toBe(1)
```
Comment (kelfink) [inline app/javascript/views/userDetail/UserDetail.test.js]: doesn't seem very useful unless is also finds the specific meta which disables the link.   Can you add the. specific attribute desired, to the find ?

Comment (RamuKammagani-osi) [reply app/javascript/views/userDetail/UserDetail.test.js]: Updated!

File: app/javascript/views/addUser/AddUser.test.js [hunk 2]
```diff
@@ -34,6 +34,14 @@ describe('VerifyUser', () => {
     })
   })
 
+  describe('meta tag to disable hyperlinks for the phone number', () => {
+    it('verifies the meta tag exists', () => {
+      expect(wrapper.find('meta').length).toBe(1)
+      expect(wrapper.find('meta').props().name).toBe('format-detection')
+      expect(wrapper.find('meta').props().content).toBe('telephone=no')
```
Comment (kelfink) [inline app/javascript/views/addUser/AddUser.test.js]: Ideally we just want one test (one expect) per 'it' block.  I think we can cover all of what we want to test here with a more specific find expression...
```suggestion
      expect(wrapper.find("meta[name='format-detection']").props().content).toBe('telephone=no')
```

Comment (RamuKammagani-osi) [reply app/javascript/views/addUser/AddUser.test.js]: Updated!

**Pull request merged**